extra/modules/confiant-ad-quality/src/main/java/org/prebid/server/hooks/modules/com/confiant/adquality/core/RedisScanStateChecker.java

```diff
@@ -0,0 +1,36 @@
+package org.prebid.server.hooks.modules.com.confiant.adquality.core;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+public class RedisScanStateChecker {
+
+    private boolean isScanDisabled = true;
```

Comment:
This has to be at least marked as `volatile`, as this will be accessed from multiple threads. JMM does not have any guarantees with current approach. Unsynchronized reads/writes can be just unseen(even though on modern architectures this should not be the case).